### ISSUE TITLE
Cannot Add Product |  Could not resolve 'fileService'.

### ISSUE THREAD

**Issue by danman113** [medusajs/admin#350]
Not sure if this belongs in the admin-panel or in the core medusa repo, but I'm having a problem where I basically can't add new products to the admin UI as every time I add one I get the following error:
```
Resolution path: fileService
AwilixResolutionError: Could not resolve 'fileService'.
```

![new-product-fail](https://user-images.githubusercontent.com/8608120/153554060-30f77b56-826d-486d-bd7e-60b419f49637.gif)

According to[ this core issue ](https://github.com/medusajs/medusa/issues/562) the problem is that the admin site is attempting to upload the images to a fileserver. This makes sense but:
1. There are no images being uploaded, so it shouldn't need to use the File Service
2. The error message should be more friendly. It should say something like "Cannot upload image without File Service Plugin".
3. If you do need to set up the File Service before using the admin panel, you should really add more detailed documentation on that, at least a link at the bottom of the quickstart.

Steps to reproduce:
1. Follow the quickstart guide in the main `readme.md`
2. Try to add a new product, only filling out the name and not uploading any files.
3. Attempt to add the new product

**Comment by danman113** [comment]
Is this fixed? I checked out develop and got the same error.

**Comment by olivermrbl** [comment]
Hey @danman113! This error occurs because you do not have a file service installed. Files services are responsible for the uploading and hosting of product images. We currently have a couple of guides for different providers, all can be found [here](https://docs.medusajs.com/add-plugins/spaces). If you are more into video format, we also have a YouTube guide [here](https://www.youtube.com/watch?v=6ZOxA-v6a_Y&feature=youtu.be) for setting up DigitalOcean.

**Comment by juanzgc** [comment]
An option for a fix - might be to default to a placeholder image if no File is uploaded

**Comment by danman113** [comment]
Another option would be to require images if they're actually required. I notice that the nextjs starter kit also expects an image to be there

**Comment by olivermrbl** [comment]
Solved in #488
